regolith/builders/recentcollabsbuilder.py

```diff
@@ -55,21 +55,33 @@ def construct_global_ctx(self):
         gtx["all_docs_from_collection"] = all_docs_from_collection
 
     def get_ppl_inst_info(self, id, months):
-        '''
+        """
         return a list of tuples, (c/a, people, institution, dept, last active) who has collaborated with
         the person with the given id within the given number months from today
-        args:
-            - id: str, name of person of interest
-            - months: int, number of months
-        output: ppl_names, list of (c/a, people, institution, dept, last active)
-            - c/a: categories, either collaborator ('C') or co-author ('A'). here data is pulled
-            from pub list so assuming that all are authors.
-            - people: name
-            - institution: institution
-            - dept: additional info such as email/department to distinguish people with the same name
-                    blank for now
-            - last active: blank for now
-        '''
+        Parameters
+        ----------
+        id : str
+            id of the person of interest (i.e. "sbillinge")
```

Comment:
numpy style would put a "the" at the beginning....so ``the id of the person..."

Also maybe clearer than "person of interest"  would be "person for whom the collaborator list is being built"